src/lib/handleAuth/handleAuth.ts

```diff
@@ -13,6 +16,9 @@ export async function handleAuth({
 	let url: URL | null = null;
 	switch (params.kindeAuth) {
 		case 'login':
+			if (options.post_login_redirect_url && typeof options.post_login_redirect_url == 'string') {
+				sessionStorage.setSessionItem(KEY_POST_LOGIN_URL, options.post_login_redirect_url);
+			}
```

Comment:
(The mention of session information prompted me to test two concurrent sign-ins and whether people would return to wherever they were in my app. Concurrent sign-ins don't seem to work anyway (the first sign-in's authentication flow state is lost, causing a 500), so that's not important for this PR I guess.)